src/client/datascience/editor-integration/hoverProvider.ts

```diff
@@ -67,7 +74,7 @@ export class HoverProvider implements INotebookExecutionLogger, vscode.HoverProv
         token: vscode.CancellationToken
     ): vscode.ProviderResult<vscode.Hover> {
         const timeoutHandler = async () => {
-            await sleep(100);
+            await sleep(300);
```

Comment:
Just for info, why was the change needed here? Is it slower now and was getting called too often?

Reply:
The original timeout was too aggressive, so even without my changes the tooltips would show up maybe 30% of the time because of this timeout.